src/infra/dev/api/oi.tpl.json

```diff
@@ -289,29 +289,634 @@
+                    ]
+                  },
+                  "response_types": {
+                    "type": "array",
+                    "items": {
+                      "type": "string"
+                    },
+                    "example": ["code", "token_id_token"]
+                  },
+                  "grant_types": {
+                    "type": "array",
+                    "items": {
+                      "type": "string"
+                    },
+                    "example": ["authorization_code", "implicit"]
+                  },
+                  "application_type": {
+                    "type": "string",
+                    "example": "web"
+                  },
+                  "contacts": {
+                    "type": "array",
+                    "items": {
+                      "type": "string"
+                    },
+                    "example": ["user1@example.org", "user2@example.org"]
+                  },
+                  "client_name": {
+                    "type": "string",
+                    "example": "https://example.org"
+                  },
+                  "subject_type": {
+                    "type": "string",
+                    "example": "pairwise"
+                  },
+                  "id_token_signed_response_alg": {
+                    "type": "string",
+                    "example": "RS256"
+                  },
+                  "token_endpoint_auth_method": {
+                    "type": "string",
+                    "example": "client_secret_basic"
+                  },
+                  "scope": {
+                    "type": "string",
+                    "example": "openid email profile openid_client_registration"
+                  },
+                  "organization": {
+                    "type": "string",
+                    "example": "https://organization.com"
+                  },
+                  "description": {
+                    "type": "string",
+                    "example": "Registered via OpenID Connect Dynamic Registration"
+                  },
+                  "environment": {
+                    "type": "string",
+                    "example": "ALL"
+                  },
+                  "master": {
+                    "type": "boolean",
+                    "example": false
+                  },
+                  "spid_requested_attributes": {
+                    "type": "array",
+                    "items": {
+                      "type": "string"
+                    },
+                    "description": "Custom parameter which return all SPID attributes selected, https://docs.italia.it/italia/spid/spid-regole-tecniche/it/stabile/attributi.html",
+                    "example": ["spidCode", "name", "familyName", "fiscalNumber"]
+                  }
+                }
+              }
+            }
+          },
+          "description": "Request body",
+          "required": true
+        },
```

Comment:
We also need, from the documentation:
- URI logo_uri;
- List<String> default_acr_values;